app/controllers/data_lists_controller.rb

```diff
@@ -15,7 +15,7 @@ def index
   # GET /data_lists/1
   # GET /data_lists/1.xml
   def show
-    @data_list = DataList.find_by id: params[:id]
+    @data_list = DataList.joins("inner join users on users.id = data_lists.owned_by").select("data_lists.*, users.email").find_by(id: params[:id])
```

Comment:
Yikes... don't know how I missed that. If I'm gonna fix this, what's the proper git procedure? Can we just drop this PR and have me submit a new one?